.github/workflows/create-rc-release.yaml

```
@@ -0,0 +1,64 @@
+        description: "Release version"
+        required: true
+  issue_comment:
+    types: [created]
+
+jobs:
+  release:
+    runs-on: macos-latest
+    if: contains(github.event.comment.body, '/release-rc') && startsWith(github.event.issue.title, 'Release')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)